content/en/software_catalog/manage.md

```diff
@@ -91,24 +111,13 @@ From the **Performance** tab, find the service you are investigating. In the **I
 
 The Service Context Map provides an overview of the relationships and dependencies between services and related infrastructure. Use this view to analyze the source of an issue by looking at upstream and downstream services and infrastructure.
 
-Click a service in Software Catalog to open the side panel with the following details: 
+### Linking infrastructure telemetries
 
-### Service details by views
+The `service` tag is the primary key for Software Catalog entries. It's also the smallest common unit of analysis for Datadog telemetries with [Universal Service Tagging][12]. Set the `service` tag directly on [Kubernetes pod labels][14]. By setting the `service` tag within the `tags.datadoghq.com/service` label, all pod telemetry, like metrics and logs, receives the service tag in Datadog. This is the recommended Kubernetes service label. 
+
+In comparison, setting the label on a Kubernetes service only affects metric tagging, not other telemetry. Applying [additional container labels][13] is essential for correctly tagging logs and traces, so this approach isn't recommended.
```

Comment:
I'm not sure which approach isn't recommend and would also rephrase to:
```suggestion
In comparison, setting the label on a Kubernetes service only affects metric tagging, not other telemetry. Applying [additional container labels][13] is essential for correctly tagging logs and traces, so Datadog does not recommend this approach.
```

Reply:
@crunchypistachiobutter I was wondering if we could clarify which approach is not recommended? Because I'm not sure when I read this.